.github/workflows/alpha-releases.yml

```diff
@@ -10,35 +10,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 16.x
-          cache: yarn
-      - name: install dependencies
-        run: yarn install --frozen-lockfile --non-interactive
+      - uses: ./.github/actions/setup
       - name: build
         env:
           DISABLE_SOURCE_MAPS: true
           BROCCOLI_ENV: production
-        run: yarn ember build
+        run: pnpm ember build
       - name: test
         env:
           TEST_SUITE: each-package
-        run: yarn test
+        run: pnpm test
 
   release:
     name: Tag + Release
     runs-on: ubuntu-latest
     needs: [test]
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 16.x
-          cache: yarn
-          registry-url: 'https://registry.npmjs.org'
-      - name: install dependencies
-        run: yarn install --frozen-lockfile --non-interactive
+      - uses: ./.github/actions/setup
```

Comment:
The registry-url needs to be passed to setup-node here for pushing to npm to work